lib/config/executor/network_interface.go

```diff
@@ -74,6 +75,9 @@ type ContainerNetwork struct {
 	// set of network wide links and aliases for this container on this network
 	Aliases []string `vic:"0.1" scope:"hidden" key:"aliases"`
 
+	// Level of trust configured for this network
+	TrustLevel common.TrustLevel
```

Comment:
This common.TrustLevel is used in vic-machine, portlayer and tether. Feels like it should be moved to a common package, instead of one in cmd/vic-machine.
Assume these packages are used only in vic-machine  cli.